### ISSUE TITLE
API / Fileadapter: resource response data for create folder request

### ISSUE THREAD

**Issue by dneukirchen** [joomla-projects/media-manager-improvement#89]
### Steps to reproduce the issue

Try to create a new folder:
POST /administrator/index.php?option=com_media&format=json&task=api.files&path=/
JSON BODY: {name: 'my-folder'}

### Expected result
A single resource response (not an array of items) with the data of the newly created folder:
```
{
  success: true,
  message: null, 
  messages: null, 
  data: {
    type: "folder", 
    name: "my-folder", 
    path: "/my-folder", 
    extension: "", 
    size:…
  }
}
```

### Actual result
The contents of the newly created folder. 
``` 
{success: true, message: null, messages: null, data: []}
```

**Comment by dneukirchen** [comment]
Quick and dirty solution: https://github.com/joomla-projects/media-manager-improvement/commit/7f87259eab162de41202d20d46ebc5877f1ce752

**Comment by laoneo** [comment]
This should be fixed with PR #96.
